<!-- -*- mode: XML -*- -->

<launch>
    <!-- Start gripper action controller -->
    <node name="gripper_action_controller" pkg="wubble2_robot" type="wubble_gripper_action_controller.py" output="screen" />

    <!-- Start gripper grasp action controller -->
    <node name="gripper_grasp_controller" pkg="wubble2_robot" type="wubble_gripper_grasp_controller.py" output="screen" >
        <param name="object_presence_pressure_threshold" value="1073.50"/>
        <param name="object_presence_opening_threshold" value="0.026"/>
        <param name="gripper_action_name" value="/wubble_gripper_action"/>
        <param name="posture_action_name" value="/wubble_gripper_grasp_action"/>
        <param name="grasp_query_name" value="/wubble_grasp_status"/>
    </node>
</launch>

